### PR TITLE
add all upgrade-to-1.5 jobs to configmap

### PR DESCRIPTION
**Pull request by krzyzacy**
https://github.com/kubernetes/kubernetes/issues/37717

cc @fejta

### CODE REVIEW

**Comment by k8s-oncall** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/2103)

**Comment by saad-ali** [comment]
Thanks @krzyzacy. Could you please also add these 1.5 test suites:
* Build
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-build-1.5/
* Verify
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-verify-release-1.5/
* Unit Test
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-test-go-release-1.5
* E2E - GCE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-release-1.5
* E2E - GKE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gke-release-1.5
* Slow Tests - GCE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-slow-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-slow-release-1.5
* Slow Tests - GKE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-slow-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gke-slow-release-1.5
* Serial Tests - GCE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-serial-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-serial-release-1.5
* Serial Tests - GKE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-serial-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gke-serial-release-1.5
* AWS Tests
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-aws-release-1.5
* Federation Tests - GCE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/kubernetes-federation-build-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation-release-1.5
* Alpha Tests - GCE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-alpha-features-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-alpha-features-release-1.5
* Alpha Tests - GKE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-alpha-features-release-1.5
* Sclability Tests
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-scalability-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-scalability-release-1.5
* Ingress Tests 
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-ingress-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gke-ingress-release-1.5
* Reboot Tests - GCE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-reboot-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-reboot-release-1.5
* Reboot Tests - GKE
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-reboot-release-1.5
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gke-reboot-release-1.5
* Soak Tests
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-soak-gce-1.5-deploy/
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-soak-gce-1.5-test/
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-soak-gci-gce-1.5-deploy/
  * https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-soak-gci-gce-1.5-test/

**Comment by krzyzacy** [comment]
@saad-ali done, did not sort it coz I feel like put them together will be more obvious.

**Comment by saad-ali** [comment]
Sweet. Thanks a lot. LGTM

**Comment by saad-ali** [comment]
Will leave it to @eparis for final sign off

File: mungegithub/submit-queue/deployment/kubernetes/configmap.yaml [hunk 1]
```diff
@@ -84,7 +84,73 @@ data:
...
+    ci-kubernetes-e2e-gce-slow-release-1.5,\
+    ci-kubernetes-e2e-gci-gce-slow-release-1.5,\
+    ci-kubernetes-e2e-gke-slow-release-1.5,\
+    ci-kubernetes-e2e-gci-gke-slow-release-1.5,\
+    ci-kubernetes-e2e-gce-serial-release-1.5,\
+    ci-kubernetes-e2e-gci-gce-serial-release-1.5,\
+    ci-kubernetes-e2e-gke-serial-release-1.5,\
+    ci-kubernetes-e2e-gci-gke-serial-release-1.5,\
+    ci-kubernetes-e2e-aws-release-1.5,\
+    kubernetes-federation-build-1.5,\
```
Comment (fejta) [inline mungegithub/submit-queue/deployment/kubernetes/configmap.yaml]: This job should have a ci- prefix: https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-commit.yaml#L117

Comment (krzyzacy) [inline mungegithub/submit-queue/deployment/kubernetes/configmap.yaml]: done. copy-pasted Saad's link :-)

**Comment by krzyzacy** [comment]
hummm... dig out https://github.com/kubernetes/test-infra/issues/429

and @fejta seems you outran the bot!

**Pull request merged**